src/client/pages/HLI/HLIStadspas.tsx

```diff
@@ -233,3 +241,61 @@ export default function HLIStadspas() {
     </DetailPage>
   );
 }
+
+function BlockPassButton({ blockPassURL }: { blockPassURL: string }) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        variant="secondary"
+        onClick={() => {
+          setIsModalOpen(true);
+        }}
+      >
+        Blokeer deze Stadspas
+      </Button>
+      <Modal
+        title="Weet u zeker dat u uw stadspas wilt blokkeren ?"
+        isOpen={isModalOpen}
+        showCloseButton={false}
+        actions={
+          <ActionGroup>
+            <Button
+              type="submit"
+              variant="primary"
+              onClick={() => {
+                setIsModalOpen(false);
+                if (!blockPassURL) {
+                  return;
+                }
```

Comment:
Dit is een rare plek voor no-op. Indien je geen api url hebt voor de blokkering, dan kun je beter de optie helemaal niet aanbieden.